cmd/costmodel/main.go

```diff
@@ -23,5 +24,6 @@ func main() {
 	a.Router.GET("/healthz", Healthz)
 	rootMux.Handle("/", a.Router)
 	rootMux.Handle("/metrics", promhttp.Handler())
-	klog.Fatal(http.ListenAndServe(":9003", errors.PanicHandlerMiddleware(rootMux)))
+	handler := cors.AllowAll().Handler(rootMux)
+	klog.Fatal(http.ListenAndServe(":9003", errors.PanicHandlerMiddleware(handler)))
```

Comment:
The UI served on `localhost:1234` requesting `localhost:9090` for the API throws a CORS error unless we supply the headers.